### ISSUE TITLE
Vagrant nfs mounts break installation of dlite machine

### ISSUE THREAD

**Issue by onnimonni** [nlf/dlite#176]
**TL;DR:** Dlite won't install If I'm using vagrant with nfs mount simultaneously with dlite.

**Steps to reproduce**
1. Start vagrant box with nfs mount
2. Install and start dlite machine
3. Notice that dlite isn't mounting `/var/run/docker.sock`
4. Notice that It doesn't return ip address with: `$ dlite ip`

**Conflicting settings in `/etc/exports`**
For example vagrant creates these in my `/etc/exports`:

```
# VAGRANT-BEGIN: 501 93054459-bc2f-4df1-aee0-3f4addf71ed9
"/Users/onnimonni/dev-root/wpbox/nginx/sites-available" 192.168.34.10 -alldirs -mapall=501:20
"/Users/onnimonni/dev-root/wpbox/www" 192.168.34.10 -alldirs -mapall=501:20
# VAGRANT-END: 501 93054459-bc2f-4df1-aee0-3f4addf71ed9
```

If I shutdown vagrant and remove those lines from `/etc/exports` dlite will start correctly and then dlite adds these lines:

```
# BEGIN: dlite
/Users -network 192.168.64.0 -mask 255.255.255.0 -alldirs -maproot=root:wheel
# END: dlite
```

If I install dlite first and then start vagrant both will work correctly and I will have settings like this:

```
$ cat /etc/exports
# BEGIN: dlite
/Users -network 192.168.64.0 -mask 255.255.255.0 -alldirs -maproot=root:wheel
# END: dlite
# VAGRANT-BEGIN: 501 93054459-bc2f-4df1-aee0-3f4addf71ed9
"/Users/onnimonni/dev-root/wpbox/nginx/sites-available" 192.168.34.10 -alldirs -mapall=501:20
"/Users/onnimonni/dev-root/wpbox/www" 192.168.34.10 -alldirs -mapall=501:20
# VAGRANT-END: 501 93054459-bc2f-4df1-aee0-3f4addf71ed9
```

Is it possible to use other nfs mounts when using dlite? Anything I could look for?

This is problematic because many of our employees are using vagrant at the moment and we want to shift into docker and dlite but still we would need to use older dev tools simultaneously for a while.

**Comment by nlf** [comment]
yup, known issue, see https://github.com/nlf/dlite#troubleshooting

it's on my radar to see if i can determine if there are conflicts but i'm pretty limited in what action i can take if there are. printing a warning is better than nothing though.

good tip that starting dlite first seems to work, i'll see if adding the export for dlite to the top of the file works more reliably

**Comment by onnimonni** [comment]
Actually just to note it here:

I noticed that even when starting dlite first doesn't work consistently. So as a workaround we just moved our vagrant development outside of `/Users/` path to `/opt/vagrant` so now nfs doesn't have colliding paths for us anymore and it works correctly.
